scripts/integration/postgres/test.yaml

```diff
@@ -1,5 +1,6 @@
 features:
 - postgresql_metrics-integration-tests
+- postgres-integration-tests
```

Comment:
this name it is not so consistent with the `postgresql_metrics-integration-tests` feature, I'm open to suggestions here